src/openforms/js/components/admin/form_design/FormMetaFields.js

```diff
@@ -134,6 +136,16 @@ const FormMetaFields = ({
                 />
             </FormRow>
 
+            <FormRow>
+                <AuthPluginAutoLoginField
+                    availablePlugins={availableAuthPlugins}
+                    selectedPlugins={selectedAuthPlugins}
+                    selectedPlugin={autoLoginAuthenticationBackend}
+                    onChange={onChange}
+                    name="form.autoLoginAuthenticationBackend"
+                />
+            </FormRow>
```

Comment:
I'd actually rather see the `Field` component lifted from `AuthPluginAutoLoginField` to here, just like line 120-128. And then instead of `TextInput`, you use `AuthPluginAutoLoginField`. The `AuthPluginField` approach is old and not how we want to do/compose things, it's a bad example.